native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -25,79 +25,460 @@
 
 #include <thread>
 
+#include "base/memory/Memory.h"
 #include "platform/android/AndroidPlatform.h"
+#include "platform/BasePlatform.h"
 #include "platform/java/jni/glue/JniNativeGlue.h"
 #include "platform/java/modules/Accelerometer.h"
 #include "platform/java/modules/Battery.h"
 #include "platform/java/modules/Network.h"
 #include "platform/java/modules/SystemWindow.h"
 #include "platform/java/modules/Vibrator.h"
+#include "cocos/bindings/event/CustomEventTypes.h"
 #include "modules/Screen.h"
 #include "modules/System.h"
+#include "java/jni/log.h"
+#include "java/jni/JniHelper.h"
 
-namespace cc {
-AndroidPlatform::AndroidPlatform() {
-    _jniNativeGlue = JNI_NATIVE_GLUE();
-}
+#include "game-activity/GameActivity.cpp"
+#include "game-text-input/GameTextInput.cpp"
```

Comment:
`cpp`, `c` files need  to be marked as source file in CMakeLists.txt.

Reply:
use GameActivityInclude.cpp to include these files, CMakeLists.txt can not get the absoulte path of GameActivity.cpp